install/kubernetes/helm/istio-remote/charts/security/templates/_helpers.tpl

```diff
@@ -0,0 +1,27 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I'm trying to remove this helper from the code - I think it impedes readability and can result in subtle bugs. 
The name of the service account and labels should be well defined.

Reply:
c&p.